Build.cs

```diff
@@ -462,13 +484,16 @@ private void MakeROM(string InFile, string FileName, BackgroundWorker worker)
             worker.ReportProgress(89, "Writing spoiler log...");
```

Comment:
I think everything before this line should be wrapped in the `if (Settings.GenerateROM) {` if block. In fact maybe move WriteSpoilerLog to be the first thing in this function, then the rest is wrapped in GenerateROM

Reply:
Actually move WriteSpoilerLog out of this function. It doesn't belong here. This is the MakeROM function, not the MakeEverything function.

Reply:
I was wondering about that too. I'll ask Kjelli about it since he wrote this bit.